### PR TITLE
feature/create pdf uploader part

### PR DESCRIPTION
**Pull request by AryanSMD**

### CODE REVIEW

File: src/components/BrancheView.vue [hunk 1]
```diff
@@ -646,7 +660,27 @@ export default {
...
       });
     },
-
+    async downloadImage(fileId){
+      let result = await this.downloadImageFile(fileId);
+      let blob = new Blob([result.data], {type: 'image/*'});
+      let url = URL.createObjectURL(blob);
+      this.$refs.branchImage.src = url;
+    },
+    async downloadDocument(file){
```
Comment (mzarsaw) [inline src/components/BrancheView.vue:666]: please use httpClient instead of using Axios directly

Comment (AryanSMD) [reply src/components/BrancheView.vue:666]: done

File: src/httpClient.js [hunk 2]
```diff
@@ -43,7 +43,10 @@ requestFetch.interceptors.response.use((response) => {
     }
     else {
         store.dispatch('loader/hide')
-        store.dispatch('error/displayErrorMessage', 'An error occurred. Please try again later.');
+        store.dispatch('error/displayErrorMessage', 'Your session has expired!');
```
Comment (mzarsaw) [inline src/httpClient.js:43]: please show this message when session expired

**Review by mzarsaw — CHANGES_REQUESTED** [review]
please fix httpClient issue

File: src/httpClient.js (hunk 2 above)
Comment (AryanSMD) [reply src/httpClient.js:43]: done. but response.status for this feature is 0 not 401

**Pull request merged**